app/controllers/bacons_controller.rb

```diff
@@ -66,6 +66,8 @@ def stats
       { value: 0.1, color: 'yellow' },
       { value: 0.0, color: 'green' }
     ]
+
+    render json: @by_launches if request.format.symbol == :json
```

Comment:
Does this sometimes render JSON and sometimes HTML?

If so, this is the more common Rails way to say it:

http://stackoverflow.com/a/20188230/3005
```ruby
respond_to do |format|
  format.html # renders the matching template
  format.json { render json: @by_launches }
end
```